In [1]:
import numpy as np
import pandas as pd
#Visualización
from pandas.plotting import scatter_matrix
import matplotlib.pyplot as plt
import seaborn as sns

url_fichero_ene18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/b7e5a71c-7e61-4488-bc38-8f6824ecae96/download'
url_fichero_feb18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/e47d1426-fb66-45e6-a2a7-2fbdcb65486a/download'
url_fichero_mar18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/0f553982-d3d1-47e8-a7bb-e4c40a84d545/download'
url_fichero_abr18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/8f316266-c701-4ba5-bcfa-3a208ddee531/download'
url_fichero_may18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/99fcf14b-aed6-45ca-bcfe-ec9cd3707fa6/download'
url_fichero_jun18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/9e54810a-ee46-45ff-a422-de90a53c7394/download'
url_fichero_jul18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/c4b54aa5-95d5-4fe7-a02f-33b2536fc59a/download'
url_fichero_ago18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/81ad7783-d385-4079-a54b-80e5c31fa98c/download'
url_fichero_sep18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/a1ea4062-2e06-4e4a-943e-ba43e95dab05/download'
url_fichero_oct18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/f4f99979-6877-488c-975e-d91d9189adb3/download'
url_fichero_nov18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/28b082c6-ab33-4ea1-88c8-317d021b52b7/download'
url_fichero_dic18 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/504c15d4-6d6a-467f-8456-97f0f37a47f3/download'
url_fichero_ene19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/2b3e60da-749e-4d73-88a8-8ad24fbd7031/download'
url_fichero_feb19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/3df495d5-78a2-4aaf-9b2f-b242d90faaf8/download'
url_fichero_mar19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/5ef10d30-5a55-4fdf-ac72-db2cfd93e3fe/download'
url_fichero_abr19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/ecac7ff8-f695-4671-b64a-78d22c9cf013/download'
url_fichero_may19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/170fd277-9591-4c7d-a179-79db96794a60/download'
url_fichero_jun19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/9f8f43ad-fbd6-405d-9ff5-32a4b2c4ef77/download'
url_fichero_jul19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/8064d78c-2250-4ccd-9836-1f1937c4ebc3/download'
url_fichero_ago19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/33cc9657-f922-495b-8506-1692eca11342/download'
url_fichero_sep19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/a11897f6-28e5-4bde-b33d-fd71fea273d8/download'
url_fichero_oct19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/2293b585-1185-46bd-a8cd-c68150dd78f9/download'
url_fichero_nov19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/324dc70d-fcce-45de-ae1e-62b05840b1f6/download'
url_fichero_dic19 = 'https://opendata-ajuntament.barcelona.cat/data/dataset/1dffc2aa-882e-4765-bb98-9f77e1b21d4a/resource/eefbeab6-4e89-4eb3-b54c-6ede2bd450d5/download'

dfTrafEne18 = pd.read_csv(url_fichero_ene18)
dfTrafFeb18 = pd.read_csv(url_fichero_feb18)
dfTrafMar18 = pd.read_csv(url_fichero_mar18)
dfTrafAbr18 = pd.read_csv(url_fichero_abr18)
dfTrafMay18 = pd.read_csv(url_fichero_may18)
dfTrafJun18 = pd.read_csv(url_fichero_jun18)
dfTrafJul18 = pd.read_csv(url_fichero_jul18)
dfTrafAgo18 = pd.read_csv(url_fichero_ago18)
dfTrafSep18 = pd.read_csv(url_fichero_sep18)
dfTrafOct18 = pd.read_csv(url_fichero_oct18)
dfTrafNov18 = pd.read_csv(url_fichero_nov18)
dfTrafDic18 = pd.read_csv(url_fichero_dic18)
dfTrafEne19 = pd.read_csv(url_fichero_ene19)
dfTrafFeb19 = pd.read_csv(url_fichero_feb19)
dfTrafMar19 = pd.read_csv(url_fichero_mar19)
dfTrafAbr19 = pd.read_csv(url_fichero_abr19)
dfTrafMay19 = pd.read_csv(url_fichero_may19)
dfTrafJun19 = pd.read_csv(url_fichero_jun19)
dfTrafJul19 = pd.read_csv(url_fichero_jul19)
dfTrafAgo19 = pd.read_csv(url_fichero_ago19)
dfTrafSep19 = pd.read_csv(url_fichero_sep19)
dfTrafOct19 = pd.read_csv(url_fichero_oct19)
dfTrafNov19 = pd.read_csv(url_fichero_nov19)
dfTrafDic19 = pd.read_csv(url_fichero_dic19)

dfTrafBarcelona18 = pd.concat([dfTrafEne18,dfTrafFeb18,dfTrafMar18,dfTrafAbr18,dfTrafMay18,dfTrafJun18,dfTrafJul18,dfTrafAgo18,dfTrafSep18,dfTrafOct18,dfTrafNov18,dfTrafDic18])
dfTrafBarcelona19 = pd.concat([dfTrafEne19,dfTrafFeb19,dfTrafMar19,dfTrafAbr19,dfTrafMay19,dfTrafJun19,dfTrafJul19,dfTrafAgo19,dfTrafSep19,dfTrafOct19,dfTrafNov19,dfTrafDic19])

dfTrafBarcelona18_19 = pd.concat([dfTrafBarcelona18,dfTrafBarcelona19])

print(dfTrafBarcelona18_19)

        idTram  infoDisponible            data  tempsActual  tempsPrevist  \
0            1               1  20180101001553          300           299   
1            2               1  20180101001553          256           255   
2            3               1  20180101001553          304           304   
3            4               1  20180101001553          324           324   
4            5               1  20180101001553            0             0   
...        ...             ...             ...          ...           ...   
455437      74               0  20191231235553           69             0   
455438      75               0  20191231235553          202             0   
455439      76               0  20191231235553           80             0   
455440      77               1  20191231235553          430             0   
455441      78               1  20191231235553          425             0   

        tempsRecorregutFutur  factorReferenciaActual  tendencia  
0        

In [2]:
print('dfTrafEne18: ',dfTrafEne18.shape)
print('dfTrafFeb18: ',dfTrafFeb18.shape)
print('dfTrafMar18: ',dfTrafMar18.shape)
print('dfTrafAbr18: ',dfTrafAbr18.shape)
print('dfTrafMay18: ',dfTrafMay18.shape)
print('dfTrafJun18: ',dfTrafJun18.shape)
print('dfTrafJul18: ',dfTrafJul18.shape)
print('dfTrafAgo18: ',dfTrafAgo18.shape)
print('dfTrafSep18: ',dfTrafSep18.shape)
print('dfTrafOct18: ',dfTrafOct18.shape)
print('dfTrafNov18: ',dfTrafNov18.shape)
print('dfTrafDic18: ',dfTrafDic18.shape)
print('dfTrafEne19: ',dfTrafEne19.shape)
print('dfTrafFeb19: ',dfTrafFeb19.shape)
print('dfTrafMar19: ',dfTrafMar19.shape)
print('dfTrafAbr19: ',dfTrafAbr19.shape)
print('dfTrafMay19: ',dfTrafMay19.shape)
print('dfTrafJun19: ',dfTrafJun19.shape)
print('dfTrafJul19: ',dfTrafJul19.shape)
print('dfTrafAgo19: ',dfTrafAgo19.shape)
print('dfTrafSep19: ',dfTrafSep19.shape)
print('dfTrafOct19: ',dfTrafOct19.shape)
print('dfTrafNov19: ',dfTrafNov19.shape)
print('dfTrafDic19: ',dfTrafDic19.shape)

dfTrafEne18:  (122262, 8)
dfTrafFeb18:  (110782, 8)
dfTrafMar18:  (119720, 8)
dfTrafAbr18:  (116686, 8)
dfTrafMay18:  (114718, 8)
dfTrafJun18:  (123574, 8)
dfTrafJul18:  (136776, 8)
dfTrafAgo18:  (139646, 8)
dfTrafSep18:  (21566, 8)
dfTrafOct18:  (82, 8)
dfTrafNov18:  (82, 8)
dfTrafDic18:  (82, 8)
dfTrafEne19:  (82, 8)
dfTrafFeb19:  (82, 8)
dfTrafMar19:  (82, 8)
dfTrafAbr19:  (15088, 8)
dfTrafMay19:  (699616, 8)
dfTrafJun19:  (668304, 8)
dfTrafJul19:  (689598, 8)
dfTrafAgo19:  (692952, 8)
dfTrafSep19:  (669864, 8)
dfTrafOct19:  (680550, 8)
dfTrafNov19:  (442884, 8)
dfTrafDic19:  (455442, 8)


Vemos que los meses desde Septiembre 2018 hasta Abril 2019 disponen de muy pocos datos

In [3]:
dfTrafBarcelona18

idTram  infoDisponible            data  tempsActual  tempsPrevist  \
0        1               1  20180101001553          300           299   
1        2               1  20180101001553          256           255   
2        3               1  20180101001553          304           304   
3        4               1  20180101001553          324           324   
4        5               1  20180101001553            0             0   
..     ...             ...             ...          ...           ...   
77      78               1  20181201001051            0             0   
78    2001               1  20181201001051            0             0   
79    2002               1  20181201001051            0             0   
80    2003               1  20181201001051            0             0   
81    2004               1  20181201001051            0             0   

    tempsRecorregutFutur  factorReferenciaActual  tendencia  
0                      1                       1          1  
1                      1                       1          1  
2                      1                       1          1  
3                      1                       1          1  
4                     -1                      -1         -1  
..                   ...                     ...        ...  
77                    -1                      -1         -1  
78                    -1                      -1         -1  
79                    -1                      -1         -1  
80                    -1                      -1         -1  
81                    -1                      -1         -1  

[1005976 rows x 8 columns]

In [4]:
dfTrafBarcelona19

idTram  infoDisponible            data  tempsActual  tempsPrevist  \
0            1               1  20190101000551            0             0   
1            2               1  20190101000551            0             0   
2            3               1  20190101000551          304           306   
3            4               1  20190101000551            0             0   
4            5               1  20190101000551            0             0   
...        ...             ...             ...          ...           ...   
455437      74               0  20191231235553           69             0   
455438      75               0  20191231235553          202             0   
455439      76               0  20191231235553           80             0   
455440      77               1  20191231235553          430             0   
455441      78               1  20191231235553          425             0   

        tempsRecorregutFutur  factorReferenciaActual  tendencia  
0                         -1                      -1         -1  
1                         -1                      -1         -1  
2                          1                       1          1  
3                         -1                      -1         -1  
4                         -1                      -1         -1  
...                      ...                     ...        ...  
455437                     1                      -1         -1  
455438                     1                      -1         -1  
455439                     1                      -1         -1  
455440                     1                      -1         -1  
455441                     1                      -1         -1  

[5014544 rows x 8 columns]

In [5]:
dfTrafBarcelona18_19.dtypes

idTram                    int64
infoDisponible            int64
data                      int64
tempsActual               int64
tempsPrevist              int64
tempsRecorregutFutur      int64
factorReferenciaActual    int64
tendencia                 int64
dtype: object

In [6]:
dfTrafBarcelona18_19.describe()

idTram  infoDisponible          data   tempsActual  tempsPrevist  \
count  6.020520e+06    6.020520e+06  6.020520e+06  6.020520e+06  6.020520e+06   
mean   6.325742e+01    7.466930e-01  2.018911e+13  1.753134e+02  1.735205e+02   
std    2.158060e+02    4.349053e-01  3.866643e+09  1.845301e+02  1.839650e+02   
min    1.000000e+00    0.000000e+00  2.018010e+13  0.000000e+00  0.000000e+00   
25%    2.000000e+01    0.000000e+00  2.019052e+13  0.000000e+00  0.000000e+00   
50%    4.000000e+01    1.000000e+00  2.019073e+13  1.640000e+02  1.640000e+02   
75%    6.000000e+01    1.000000e+00  2.019100e+13  2.420000e+02  2.400000e+02   
max    2.004000e+03    1.000000e+00  2.019123e+13  6.372000e+03  3.003000e+03   

       tempsRecorregutFutur  factorReferenciaActual     tendencia  
count          6.020520e+06            6.020520e+06  6.020520e+06  
mean           3.746347e-01            3.573930e-01  3.580295e-01  
std            9.458795e-01            9.475746e-01  9.391897e-01  
min           -1.000000e+00           -1.000000e+00 -1.000000e+00  
25%           -1.000000e+00           -1.000000e+00 -1.000000e+00  
50%            1.000000e+00            1.000000e+00  1.000000e+00  
75%            1.000000e+00            1.000000e+00  1.000000e+00  
max            2.000000e+00            2.000000e+00  2.000000e+00

In [7]:
dfTrafBarcelona18_19['tempsActual'].value_counts()

0       1881268
164      137313
260      123346
104       89472
189       74770
         ...   
1383          1
1379          1
1373          1
1372          1
3975          1
Name: tempsActual, Length: 1507, dtype: int64

In [8]:
dfTrafBarcelona18_19 = dfTrafBarcelona18_19.reset_index()
print(dfTrafBarcelona18_19)

          index  idTram  infoDisponible            data  tempsActual  \
0             0       1               1  20180101001553          300   
1             1       2               1  20180101001553          256   
2             2       3               1  20180101001553          304   
3             3       4               1  20180101001553          324   
4             4       5               1  20180101001553            0   
...         ...     ...             ...             ...          ...   
6020515  455437      74               0  20191231235553           69   
6020516  455438      75               0  20191231235553          202   
6020517  455439      76               0  20191231235553           80   
6020518  455440      77               1  20191231235553          430   
6020519  455441      78               1  20191231235553          425   

         tempsPrevist  tempsRecorregutFutur  factorReferenciaActual  tendencia  
0                 299                     1           

Quitamos las filas que contienen valor '0' de Tiempo Actual.

In [9]:
dfTrafBarcelona18_19 = dfTrafBarcelona18_19.drop(dfTrafBarcelona18_19[dfTrafBarcelona18_19['tempsActual']==0].index)
print(dfTrafBarcelona18_19)

          index  idTram  infoDisponible            data  tempsActual  \
0             0       1               1  20180101001553          300   
1             1       2               1  20180101001553          256   
2             2       3               1  20180101001553          304   
3             3       4               1  20180101001553          324   
5             5       6               1  20180101001553          275   
...         ...     ...             ...             ...          ...   
6020515  455437      74               0  20191231235553           69   
6020516  455438      75               0  20191231235553          202   
6020517  455439      76               0  20191231235553           80   
6020518  455440      77               1  20191231235553          430   
6020519  455441      78               1  20191231235553          425   

         tempsPrevist  tempsRecorregutFutur  factorReferenciaActual  tendencia  
0                 299                     1           

Eliminamos las columnas que no nos interesan para nuestro análisis

In [10]:
dfTrafBarcelona18_19 = dfTrafBarcelona18_19.drop(columns=['index','idTram','infoDisponible','tempsPrevist','tempsRecorregutFutur','factorReferenciaActual','tendencia'])
print(dfTrafBarcelona18_19)

                   data  tempsActual
0        20180101001553          300
1        20180101001553          256
2        20180101001553          304
3        20180101001553          324
5        20180101001553          275
...                 ...          ...
6020515  20191231235553           69
6020516  20191231235553          202
6020517  20191231235553           80
6020518  20191231235553          430
6020519  20191231235553          425

[4139252 rows x 2 columns]


Ahora vamos a convertir el campo 'data' en formato Fecha para convertirlo en Indice

In [11]:
dfTrafBarcelona18_19['data'] = pd.to_datetime(dfTrafBarcelona18_19['data'], format='%Y%m%d%H%M%S')
print(dfTrafBarcelona18_19)

                       data  tempsActual
0       2018-01-01 00:15:53          300
1       2018-01-01 00:15:53          256
2       2018-01-01 00:15:53          304
3       2018-01-01 00:15:53          324
5       2018-01-01 00:15:53          275
...                     ...          ...
6020515 2019-12-31 23:55:53           69
6020516 2019-12-31 23:55:53          202
6020517 2019-12-31 23:55:53           80
6020518 2019-12-31 23:55:53          430
6020519 2019-12-31 23:55:53          425

[4139252 rows x 2 columns]


In [12]:
dfTrafBarcelona18_19 = dfTrafBarcelona18_19.set_index('data')
print(dfTrafBarcelona18_19)

                     tempsActual
data                            
2018-01-01 00:15:53          300
2018-01-01 00:15:53          256
2018-01-01 00:15:53          304
2018-01-01 00:15:53          324
2018-01-01 00:15:53          275
...                          ...
2019-12-31 23:55:53           69
2019-12-31 23:55:53          202
2019-12-31 23:55:53           80
2019-12-31 23:55:53          430
2019-12-31 23:55:53          425

[4139252 rows x 1 columns]


In [13]:
dfTrafBarcelona18_19 = dfTrafBarcelona18_19.resample('D').median()
pd.set_option("display.max_rows", None)
print(dfTrafBarcelona18_19)

            tempsActual
data                   
2018-01-01        213.0
2018-01-02        219.0
2018-01-03        216.0
2018-01-04        219.0
2018-01-05        212.0
2018-01-06        215.0
2018-01-07        212.0
2018-01-08        219.0
2018-01-09        219.0
2018-01-10        219.0
2018-01-11        219.0
2018-01-12        220.0
2018-01-13        213.0
2018-01-14        214.0
2018-01-15        220.0
2018-01-16        220.0
2018-01-17        219.0
2018-01-18        220.0
2018-01-19        220.0
2018-01-20        212.0
2018-01-21        211.0
2018-01-22        219.0
2018-01-23        220.0
2018-01-24        220.0
2018-01-25        221.0
2018-01-26        225.0
2018-01-27        219.0
2018-01-28        219.0
2018-01-29        220.0
2018-01-30        220.0
2018-01-31        221.0
2018-02-01        222.0
2018-02-02        221.0
2018-02-03        219.0
2018-02-04        217.0
2018-02-05        221.0
2018-02-06        219.0
2018-02-07        221.0
2018-02-08        220.0
2018-02-09      

In [14]:
dfTrafBarcelona18_19.isnull().sum()

tempsActual    229
dtype: int64

Hay 229 días sin datos de tráfico. Se trata de los datos desde el 06 de septiembre 2018 hasta el 29 de abril 2019, como se vio antes.

In [15]:
dfTrafBarcelona18_19['2018-09-01':'2018-09-30'] = dfTrafBarcelona18_19['2018-09-01':'2018-09-30'].fillna({'tempsActual':dfTrafBarcelona18_19['2018-09-01':'2018-09-30']['tempsActual'].mean()})
dfTrafBarcelona18_19['2018-10-01':'2018-10-31'] = dfTrafBarcelona18_19['2018-10-01':'2018-10-31'].fillna({'tempsActual':dfTrafBarcelona18_19['2018-10-01':'2018-10-31']['tempsActual'].mean()})
dfTrafBarcelona18_19['2018-11-01':'2018-11-30'] = dfTrafBarcelona18_19['2018-11-01':'2018-11-30'].fillna({'tempsActual':dfTrafBarcelona18_19['2018-11-01':'2018-11-30']['tempsActual'].mean()})
dfTrafBarcelona18_19['2018-12-01':'2018-12-31'] = dfTrafBarcelona18_19['2018-12-01':'2018-12-31'].fillna({'tempsActual':dfTrafBarcelona18_19['2018-12-01':'2018-12-31']['tempsActual'].mean()})
dfTrafBarcelona18_19['2019-01-01':'2019-01-31'] = dfTrafBarcelona18_19['2019-01-01':'2019-01-31'].fillna({'tempsActual':dfTrafBarcelona18_19['2019-01-01':'2019-01-31']['tempsActual'].mean()})
dfTrafBarcelona18_19['2019-02-01':'2019-02-28'] = dfTrafBarcelona18_19['2019-02-01':'2019-02-28'].fillna({'tempsActual':dfTrafBarcelona18_19['2019-02-01':'2019-02-28']['tempsActual'].mean()})
dfTrafBarcelona18_19['2019-03-01':'2019-03-31'] = dfTrafBarcelona18_19['2019-03-01':'2019-03-31'].fillna({'tempsActual':dfTrafBarcelona18_19['2019-03-01':'2019-03-31']['tempsActual'].mean()})
dfTrafBarcelona18_19['2019-04-01':'2019-04-30'] = dfTrafBarcelona18_19['2019-04-01':'2019-04-30'].fillna({'tempsActual':dfTrafBarcelona18_19['2019-04-01':'2019-04-30']['tempsActual'].mean()})

pd.set_option("display.max_rows", None)
print(dfTrafBarcelona18_19)

            tempsActual
data                   
2018-01-01        213.0
2018-01-02        219.0
2018-01-03        216.0
2018-01-04        219.0
2018-01-05        212.0
2018-01-06        215.0
2018-01-07        212.0
2018-01-08        219.0
2018-01-09        219.0
2018-01-10        219.0
2018-01-11        219.0
2018-01-12        220.0
2018-01-13        213.0
2018-01-14        214.0
2018-01-15        220.0
2018-01-16        220.0
2018-01-17        219.0
2018-01-18        220.0
2018-01-19        220.0
2018-01-20        212.0
2018-01-21        211.0
2018-01-22        219.0
2018-01-23        220.0
2018-01-24        220.0
2018-01-25        221.0
2018-01-26        225.0
2018-01-27        219.0
2018-01-28        219.0
2018-01-29        220.0
2018-01-30        220.0
2018-01-31        221.0
2018-02-01        222.0
2018-02-02        221.0
2018-02-03        219.0
2018-02-04        217.0
2018-02-05        221.0
2018-02-06        219.0
2018-02-07        221.0
2018-02-08        220.0
2018-02-09      

Exportamos a un fichero csv el dataframe trabajado resultante

In [17]:
dfTrafBarcelona18_19.to_csv('DatosTraficoBarcelona.csv')